packages/mobile/src/utils/populateCoverArtSizes.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+  Collection,
+  CoverArtSizes,
+  Track,
+  UserMetadata
+} from '@audius/common'
+import { DefaultSizes, SquareSizes } from '@audius/common'
+
+import { audiusBackendInstance } from 'app/services/audius-backend-instance'
+
+export const populateCoverArtSizes = async <
+  T extends Pick<Track | Collection, 'cover_art_sizes' | 'cover_art'> & {
```

Comment:
niceee, i might make that a helper type just to keep the inline def from getting too wild, nit tho.

Reply:
Yeah also I wasn't sure if I should just write the type explicitly instead of using pick. Either works fine, but not sure if one is more maintainable.